source/cli-implementation.js

```diff
@@ -37,6 +37,7 @@ const cli = meow(`
 	  --test-script          Name of npm run script to run tests before publishing (default: test)
 	  --no-2fa               Don't enable 2FA on new packages (not recommended)
 	  --message              Version bump commit message, '%s' will be replaced with version (default: '%s' with npm and 'v%s' with yarn)
+	  --publish-scoped       When publishing a scoped package, you need to provide this option to force publish it publically
```

Comment:
This would only be required if you do the first ever version non-interactively. The interactive UI has handling for scoped package and asks you whether you want it publicly. I feel like this is too much of an edge-case to have a flag for. I would rather just document in the readme that the first ever version has to be done interactively.